packages/varlock-docs-mcp/.gitignore

```diff
@@ -0,0 +1,176 @@
+# Logs
```

Comment:
Seems like a very general and very wide gitignore. Many things likely already covered in our root gitignore. Not a huge deal but could slim it down...